ui/src/app/view/timestamp-table/timestamp-table.component.html

```diff
@@ -122,50 +122,52 @@
           style="width: 15%"
-          [ngClass]="timestamp.outdatedMessage ?  'disabledRow' : timestamp.uiReadByUser? 'timestampContent' : 'unreadTimestamp'  ">
+          [ngClass]="'timestampContent'">
         {{(timestamp.logOfTimestamp | timestampToTimezone: timestamp.portOfCall: ports)[0]}}
       </td>
       <td pTooltip ="UTC: {{(timestamp.eventDateTime | timestampToTimezone: timestamp.portOfCall: ports)[1]}}"
           style="width: 15%"
-          [ngClass]="timestamp.outdatedMessage ?  'disabledRow' : timestamp.uiReadByUser? 'timestampContent' : 'unreadTimestamp'  ">
+          [ngClass]="'timestampContent'">
         {{(timestamp.eventDateTime | timestampToTimezone: timestamp.portOfCall: ports)[0]}}
       </td>
         <!--
       <td
-        [ngClass]="timestamp.outdatedMessage ?  'disabledRow' : timestamp.uiReadByUser? 'timestampContent' : 'unreadTimestamp'  ">
+        [ngClass]="'timestampContent'">
         <div
-          [ngClass]="timestamp.outdatedMessage ?  'disabledRow' : timestamp.uiReadByUser? 'timestampContent' : 'unreadTimestamp'  ">{{ (timestamp.portPrevious | portIdToPort: ports).unLocode}}</div>
+          [ngClass]="'timestampContent'">{{ (timestamp.portPrevious | portIdToPort: ports).unLocode}}</div>
         <div class="rowDetails">{{ (timestamp.portPrevious | portIdToPort: ports).name}}</div>
       </td>
        -->
       <td
-        [ngClass]="timestamp.outdatedMessage ?  'disabledRow' : timestamp.uiReadByUser? 'timestampContent' : 'unreadTimestamp'  "
+        [ngClass]="'timestampContent'"
         style="width: 8%">
         <div
-          [ngClass]="timestamp.outdatedMessage ?  'disabledRow' : timestamp.uiReadByUser? 'timestampContent' : 'unreadTimestamp'  ">{{ (timestamp.portOfCall | portIdToPort: ports).unLocode}}</div>
+          [ngClass]="'timestampContent'">{{ (timestamp.portOfCall | portIdToPort: ports).unLocode}}</div>
         <div class="rowDetails">{{timestamp.portOfCall?.name}}</div>
       </td>
       <td style="width: 6%"
-          [ngClass]="timestamp.outdatedMessage ?  'disabledRow' : timestamp.uiReadByUser? 'timestampContent' : 'unreadTimestamp'  ">
-        <!--   somehow there is a porblem acessing the terminals   -->
-        <div *ngIf="timestamp.facilitySMDGCode!= null">
-          <div pTooltip="{{timestamp.facilitySMDGCode}}" tooltipPosition="top"
-               tooltipStyleClass="ToolTipClass">{{timestamp.facilitySMDGCode}}</div>
+        [ngClass]="'timestampContent'">
+          <div pTooltip="{{timestamp.eventLocation?.facilityCode}}" tooltipPosition="top" tooltipStyleClass="ToolTipClass">
+            {{timestamp.eventLocation?.facilityCode}}</div>
+        
+        <div *ngIf="timestamp.eventLocation?.facilityCode == null && timestamp.facilitySMDGCode != null">
+          <div pTooltip="{{timestamp.facilitySMDGCode}}" tooltipPosition="top" tooltipStyleClass="ToolTipClass">
+            {{timestamp.facilitySMDGCode}}</div>
```

Comment:
Open question, should we show a placeholder value if there is no facility SMDG code at all?  (A la `N/A`)